tensorflow_probability/python/experimental/stats/sample_stats.py

```diff
@@ -109,6 +109,10 @@ class RunningCovariance(object):
   which includes intermediate calculations, are held in a
   `RunningCovarianceState` as returned via `initialize` and `update` method
   calls.
+
+  `RunningCovariance` is meant to serve general streaming covariance needs.
```

Comment:
I don't think this forward-reference is necessary.

Reply:
I'm leaning towards keeping it simply because it can only do more good than harm (right?)